src/main.cpp

```diff
@@ -336,24 +350,33 @@ static void list_game_ids()
 		std::cout << gameid <<std::endl;
 }
 
-static void list_worlds()
+static void list_worlds(bool print_name, bool print_path)
 {
 	std::cout << _("Available worlds:") << std::endl;
 	std::vector<WorldSpec> worldspecs = getAvailableWorlds();
-	print_worldspecs(worldspecs, std::cout);
+	print_worldspecs(worldspecs, std::cout, print_name, print_path);
 }
 
 static void print_worldspecs(const std::vector<WorldSpec> &worldspecs,
-							 std::ostream &os)
+	std::ostream &os, bool print_name, bool print_path)
 {
 	for (const WorldSpec &worldspec : worldspecs) {
 		std::string name = worldspec.name;
 		std::string path = worldspec.path;
-		if (name.find(' ') != std::string::npos)
-			name = std::string("'").append(name).append("'");
-		path = std::string("'").append(path).append("'");
-		name = padStringRight(name, 14);
-		os << "  " << name << " " << path << std::endl;
+		if (print_name && print_path) {
+			os << "\t" << name << "\t\t" << path << std::endl;
+
+			if (name.find(' ') != std::string::npos)
+				name = std::string("'").append(name).append("'");
+			path = std::string("'").append(path).append("'");
+
+			name = padStringRight(name, 14);
```

Comment:
Why do you modify `name` and `path` here without using them?

Reply:
Oops. I hadn't meant for all of those changes to be included.

That was when I was trying different things with spacing, so I was modifying the format a lot. I was keeping the code there so it was easy to switch formats for testing. 

If we want to use the tab formatting (which we were talking about above), that code can be removed. If we don't want to use it, I can undo the changes.